opendj-server-legacy/src/test/java/org/opends/server/extensions/CRAMMD5SASLMechanismHandlerTestCase.java

```diff
@@ -23,6 +23,7 @@
  *
  *      Copyright 2006-2008 Sun Microsystems, Inc.
  *      Portions Copyright 2014-2015 ForgeRock AS
+ *      Portions Copyrighted 2019 OGIS-RI Co., Ltd.v
```

Comment:
`OGIS-RI Co., Ltd.v` -> `OGIS-RI Co., Ltd.`

Reply:
Thank you for noticing, I fixed this typo.